jobQueue/jobWrapper.js

```diff
@@ -130,7 +130,7 @@ function logJobOnComplete(req, job) {
       .then((resObj) => {
         logObject.user = resObj.username;
         logObject.token = resObj.tokenname;
-      });
+      }).catch(() => {}); // no-op
```

Comment:
It looks like getTokenDetailsFromRequest throws if its call to getTokenDetailsFromTokenString throws, which happens if its call to jsonwebtoken module's jwt.verify throws, which happens if the token's signature and optional expiration, audience, or issuer are not valid.
Would there be any value in logging this in the catch here? Maybe shed some light on why we're getting these invalid tokens?